### DISCUSSION TITLE
log file formats, asc and blf

### DISCUSSION THREAD

**Discussion by JohnLussmyer** [collin80/SavvyCAN#763] [General]
I'm working with a device, and the manufacturer won't provide ANY help unless I supply a log file in "asc or blf" formats.  (and if we need to know what tool they expect to use the files with - they haven't told me.)
Are these formats that SavvyCAN can write?

**Comment by collin80** [comment]
SavvyCAN can write ASC files. ASC and BLF are file formats generated and read by Vector tools - CANAlyzer and CANoe for instance. So, chances are this manufacturer is going to use one of Vector's tools to read your log files. 

So, yes, you can use SavvyCAN to capture CAN traffic and then save it in ASC format which the Vector tools should be able to read.
